crates/db_schema/src/impls/post.rs

```diff
@@ -117,9 +117,7 @@ impl Post {
       .filter(post::local.eq(true))
       .filter(post::deleted.eq(false))
       .filter(post::removed.eq(false))
-      .filter(
-        post::published.ge(Utc::now().naive_utc() - SITEMAP_DAYS.expect("TimeDelta out of bounds")),
-      )
+      .filter(post::published.ge(Utc::now().naive_utc() - SITEMAP_DAYS.unwrap_or_default()))
```

Comment:
You can change SITEMAP_DAYS like this so its not an option:
`pub const SITEMAP_DAYS: TimeDelta = TimeDelta::days(31);`